Proofread reporting chapter (#2578)
centreon/centreon-documentation: versioned_docs/version-23.10/reporting/installation.md

#### Firmware and software layer

- OS: RHEL / Oracle Linux / Alma 8 or 9, Debian 11
- SGBD: MariaDB 10.5

> - Firewalld: Disabled ([look here](../installation/installation-of-a-central-server/using-packages.md#Configurer-ou-désactiver-le-pare-feu))

Reviewer: ```suggestion
- Firewalld: Disabled ([look here](../installation/installation-of-a-central-server/using-packages.md#configure-or-disable-the-firewall))
```

> - SELinux : Disabled ([look here](../installation/installation-of-a-central-server/using-packages.md#Désactiver-SELinux))

Reviewer: ```suggestion
- SELinux : Disabled ([look here](../installation/installation-of-a-central-server/using-packages.md#disable-selinux))
```

> - SELinux: Disabled ([look here](../installation/installation-of-a-central-server/using-packages.md#Désactiver-SELinux))

Reviewer: ```suggestion
- SELinux: Disabled ([look here](../installation/installation-of-a-central-server/using-packages.md#disable-selinux))
```

- IP/DNS of the reporting database (localhost strongly recommended)

- Access (user/password) to the reporting database
- Define and retrieve the ssh password of the centreonBI user on the Central server (for the availability of reports generated on the interface)

#### Procedure

> 1. To start installing the reporting server, install the Business repository. You can find it on the [support portal](https://support.centreon.com/hc/en-us/categories/10341239833105-Repositories).

Reviewer: ?
